frontend/src/app/decorators/auth.decorator.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from '../models/user.model';
```

Comment:
Question: Why format user is not in AuthService ?

Who will use this format function ?